app/helpers/shf_icons_helper.rb

```diff
@@ -216,9 +227,19 @@ module ShfIconsHelper
   # Try to keep these in the same order as the icon constant groups above.
   #
   METHODS_AND_ICONS = [
-      { method_name_start: 'edit', icon: FA_EDIT },
-      { method_name_start: 'view', icon: FA_VIEW },
-      { method_name_start: 'destroy', icon: FA_DESTROY },
+      { method_name_start: 'edit', icon: FA_EDIT,
+        html_options:{ class: 'edit'} },
+
+      { method_name_start: 'view', icon: FA_VIEW,
+        color: 'blue',
+        fa_style: 'far',
+        html_options:{ class: 'view'}},
+
+      { method_name_start: 'destroy', icon: FA_DESTROY,
+        color: 'red',
+        fa_style: 'far',
+        html_options:{ class: 'delete'}
```

Comment:
I don't think we should be hard-coding a class for any icon.  At first I thought it might be useful to do that for certain icons - such as the delete icon - where the class `text_danger` would result in a red icon.  However, that would only work if we also allowed the user to pass in their own class (via html_options) and merged the two class definitions.

However, you're using specific style statements for assigning a color to the icon, which is fine.

So, I recommend _not_ using any class in the icon definition, and just allow the user to pass in any desired class.  Also, classes such as `delete` etc. are too generic.  And, the user may have to assign a specific class to the icon, where the class itself is preordained and cannot be changed (such as a class associated with a JS package).

Reply:
Can easily change the 'definition' of those standard icons (edit, view, destroy) to remove the classes.
I used the 'delete' class because that was used in a view.
And yes -- a user _can_ also pass in a class definition.  Anything the user passes in is merged with the `html_options` defined for the standardized icon: `merged_html_options = default_html_options.merge(html_options)`  So anything the user passes in takes precedent and will override anything in the standard definition.  (See the spec)